### ISSUE TITLE
AD User 

### ISSUE THREAD

**Issue by Colin-988** [rustdesk/rustdesk-server-pro#438]
Hallo, im using an LDAP Bind for login in rustdesk. 
When in try to login i get this error message:
![grafik](https://github.com/user-attachments/assets/02693964-6497-45d5-a88c-76315a065e2f)

I have to delete the user and his linked devices, after that the user can login successfully.
Thx, for help :D

**Comment by rustdesk** [comment]
The error message has told the reason. There is already a user with the same email.

**Comment by Colin-988** [comment]
yeah but how is it possible when the user cant login with the ad credentials and after i delete the user its possible.

**Comment by rustdesk** [comment]
> the user

This user is logged in with the same AD account?

**Comment by Colin-988** [comment]
no, hes is logged out and i have to delete him before he can login succesfully

**Comment by rustdesk** [comment]
@fufesou

**Comment by rustdesk** [comment]
This is expected if the original user with the same email is not the same AD account.

**Comment by fufesou** [comment]
> id changed etc?

After the user is deleted, if a new account with the same email address is created, the id will change.
Deleting a user directly deletes the database data.

**Comment by Colin-988** [comment]
yeah but if nothing changed with the ad account. Normally if you login with your ldap bind, rust desk is creating an local account for that user, is that right?

**Comment by rustdesk** [comment]
Let us know if it happen again, please take a snaphost of the user before you delete it, we will then investigate why we fail to find the ldap user in our database and attempt to create a new user.

@fufesou follow it up then.
